sourcemash/frontend/templates/splash.tpl

```diff
@@ -54,7 +54,7 @@
         </div>
         <div class="col s8">
             <p class="promo-caption">Read by topic, browse by feed.</p>
-            <p class="light">Customize your news reading experience to make Sourcemash work for you. We are also always <a href="mailto:support@sourcemash.com">open to feedback</a> and can <a href="mailto:support@sourcemash.com">answer any questions</a> that you may have.</p>
+            <p class="light">Customize your news reading experience to make Sourcemash work for you. We are also always <a href="mailto:support@sourcemash.com?subject=Suggestions for Sourcemash">open to feedback</a> and can <a href="mailto:support@sourcemash.com?subject=Suggestions for Sourcemash">answer any questions</a> that you may have.</p>
```

Comment:
It's not a big deal. Just seems odd to me that we say we can answer questions via email and then have the subject line default to something they may not want to say. Like if they have a question or compliment

Alex Gerstein
gerstein.alex@gmail.com




On Thu, May 21, 2015 at 6:27 PM -0700, "Scott Gladstone" <notifications@github.com> wrote:












In sourcemash/frontend/templates/splash.tpl:
> @@ -54,7 +54,7 @@
>          </div>
>          <div class="col s8">
>              <p class="promo-caption">Read by topic, browse by feed.</p>
> -            <p class="light">Customize your news reading experience to make Sourcemash work for you. We are also always <a href="mailto:support@sourcemash.com">open to feedback</a> and can <a href="mailto:support@sourcemash.com">answer any questions</a> that you may have.</p>
> +            <p class="light">Customize your news reading experience to make Sourcemash work for you. We are also always <a href="mailto:support@sourcemash.com?subject=Suggestions for Sourcemash">open to feedback</a> and can <a href="mailto:support@sourcemash.com?subject=Suggestions for Sourcemash">answer any questions</a> that you may have.</p>

Then they will use the body of the message to convey their excitement….



—
Reply to this email directly or view it on GitHub.